web/themes/joinup/templates/form/container--stats-topic.html.twig

```diff
@@ -18,10 +18,12 @@
  */
 #}
 {% if children is not empty %}
-  <div class="stats__column">
-    <div class="stats__wrapper">
-      <span class="stats__text">{{ topic }}:</span>
-      <span class="stats__text bold">{{ children }}</span>
+  {% if children.__toString() >= 1 %}
```

Comment:
Can you explain this change?

Reply:
Well, since we changed the logic of the total downloads counter in solution_rdf_entity_view() this does not need anymore. Reverted!